Automattic/wp-calypso
# Plans: Implement money-back guarantee banner in HTML+SVG

#### Changes proposed in this Pull Request

Resolves `1164141197617539-as-1200006697536409`.

* Replace the static images for the 14-day money-back guarantee banner with HTML and SVG content, to reduce loading times and layout shift on slower devices.

#### Known issues

* Line wrapping behavior may be slightly different than exists in production, since the current banner is a static image and this PR renders all the text HTML.
* This PR is using `breakpoint-deprecated` to handle screeen size queries, when `break-small` would be more appropriate. Unfortunately I experienced build errors when I used `break-small`, but this is something I think we can come back later to fix.

#### Testing instructions

* Open any experience (Calypso Blue, Green, Jetpack Connect) in your testing environment with this PR applied.
* Visit the appropriate Plans/Pricing page.
* Verify that the 14-day money-back guarantee displays correctly on your screen and looks similar to the one that exists in production.
* Verify that as you change the size of your browser's viewport, the banner changes size accordingly.
* Verify that at a screen width of 660 pixels, the banner transitions between a horizontal layout (for wider screens) and a vertical one (for narrower screens).

#### Reference screenshots (before / after)

##### Mobile

<img height="500" src="https://user-images.githubusercontent.com/670067/110137992-1a514700-7d97-11eb-87d0-cf5b1501293f.png"> <img height="500" src="https://user-images.githubusercontent.com/670067/110141335-a9ac2980-7d9a-11eb-8a9e-a5eadff52d59.png">

##### Desktop

<img width="350" src="https://user-images.githubusercontent.com/670067/110138370-8633af80-7d97-11eb-941d-2f0fac1087c4.png"> <img width="350" src="https://user-images.githubusercontent.com/670067/110141478-ce080600-7d9a-11eb-9d8c-4287b5ce907f.png">
